packages/opencensus-instrumentation-mongodb/src/mongodb.ts

```diff
@@ -136,11 +150,11 @@ export class MongoDBPlugin extends BasePlugin {
         if (plugin.tracer.currentRootSpan && arguments.length > 0 &&
             typeof resultHandler === 'function') {
           const span = plugin.tracer.startChildSpan(
-              {name: `${this.ns}.cursor`, kind: SpanKind.SERVER});
+              `${this.ns}.cursor`, SpanKind.SERVER);
```

Comment:
Can we use the object for ```startChildSpan```function instead of passing two/three arguments?

Reply:
We can't with the current API, it has changed with the version 0.0.9